lib/helper/Puppeteer.js

```diff
@@ -255,6 +255,14 @@ class Puppeteer extends Helper {
     });
   }
 
+  /**
+   * Gets page URL including hash.
+   */
+  async _getPageUrl() {
+    const hash = await this.executeScript(() => window.location.hash);
```

Comment:
Maybe we just should simply use `window.location.href` then without calling to `page.url()`
I fear that mixing both approaches can have side effects